---
title: Sessions
description: Current list of sessions at Biodiversity Genomics Academy 2023
---

## Planned Sessions

Scheduled sessions are now open for registration. New sessions and repeated sessions will be added to this page, so please keep checking back or [sign up](signup.md) to be emailed when new/repeat sessions become available.

Detailed session contents will be added over time at this site. Instructions for joining each session will be sent to registered participants.

If there are 2 time slots available for a session, pick the one you can attend more conveniently [in your time zone](https://www.timeanddate.com/worldclock/meetingtime.html?iso=20230915&p1=224&p2=41&p3=45&p4=4&p5=37&p6=170&p7=176&p8=33&p9=240&p10=22){: target="_blank"}.

??? info "Old updates, till 2023-09-04"

    **2023-08-07 Update**  
    The response to BGA23 has been OVERWHELMING! Thank you üôè **691 booked, 424 waitlisted** ü§Ø Please keep signing up below as we are working with session leaders to add more slots (adding sessions, increasing slots per session, offering recordings with discord discussions, etc). Booked out sessions have a "Join Waitlist" button. Also do **[subscribe/follow](signup.md)** to be the first to be informed when slots open.

    **2023-08-11 Update**  
    We are notifying waitlisted participants today by email on how to book extra slots. If you have a confirmed slot already that you can no longer attend, please email tol-training at sanger.ac.uk with your order details so we can free up your slot for other waitlisted participants. Thank you!

    Sessions added/changed:

    - "Introduction to manual curation - Treeval" renamed to "Setting up the Treeval pipeline for generating evidence for manual curation", and a new session added 18 Sep 11:00-13:00
    - "Starting a comparative genome study from [CNGBdb](https://db.cngb.org/){: target="_blank"}: datasets, analysis platform, and spatial transcriptomics database" new session 13 Sep 09:00-11:00
    - "Genome profiling using [Genomescope](https://github.com/tbenavi1/genomescope2.0){: target="_blank"} new session 15 Sep 09:00-11:00
    - "[OMA](https://omabrowser.org/oma/home/){: target="_blank"} and [OMArk](https://omark.omabrowser.org/){: target="_blank"} for homology exploration and gene annotation quality control" new session 15 Sep 16:00-18:00
    - "Introduction to Manual Curation - HiC and JBrowse" new session added 19 Sep 11:00-13:00
    - "Introduction to NextFlow for Genomics" two new sessions added 20 Sep 23:00-01:00, and 26 Sep 16:00-18:00
    - "[EASEL](https://gitlab.com/PlantGenomicsLab/easel){: target="_blank"} (Efficient, Accurate, Scalable Eukaryotic modeLs): software for eukaryotic structural and functional genome annotation" new session added 29 Sep 03:00-05:00

    **2023-08-14 Update**  
    We are up to **1167 sign ups and 793 waitlisted slots**!
    
    Seeing the huge demand, some session leaders have very kindly offered webinar slots (view only + text discussions) for:

    - Assembling Genomes with the VGP-Galaxy Pipeline
    - Classifying Transposable Elements with Earl Grey TE
    - Assess genome assemblies with Merqury
    - Annotating genomes the Ensembl way - General concepts + Case study
    - Annotating genomes the Ensembl way: Hands on - From RNAseq reads to gene models

    If you signed up to a session waitlist after Friday 11 Aug, you will receive information on additional slots by the end of today.

    **2023-08-16 Update**  
    We are up to **1475 sign ups and 1018 waitlisted slots**!

    All waitlisted slots as of 4pm UTC should have received information about additional slots. After this point, the only free slots are likely to be from cancellations. A couple of new topics may be added, but no additional slots are likely to be opened for existing topics.

    To be fair to everyone, we will be asking everyone to confirm attendance for each session by next week, so please keep an eye out for those emails.

    **2023-08-25 Update**  
    We are up to **1680 sign ups and 1273 waitlisted slots**!

    Sessions added/changed:

    - "Gene prediction with [BRAKER3](https://github.com/Gaius-Augustus/BRAKER){: target="_blank"}" - new sessions added 22 Sep 08:00-10:00 and 25 Sep 12:00-14:00
    - "[OMA](https://omabrowser.org/oma/home/){: target="_blank"} and [OMArk](https://omark.omabrowser.org/){: target="_blank"} for homology exploration and gene annotation quality control" new session added 21 Sep 08:00-10:00

    **2023-08-30 Update**  
    We are up to **1807 sign ups and 1401 waitlisted slots**!

    Waitlist slots are now closed for week 1. If you are registered for a Week 1 (Sep 4-8) session, you should have received a confirmation email by now. You MUST confirm your Week 1 registration by 31st August 23:59 UTC/GMT otherwise we'll be releasing your slots to the (very long) waitlists...

    Sessions added/changed:

    - "Pangenome Assembly and Assessment using Minigraph and Bandage" new session added 21 Sep 20:00-22:00

    **2023-09-04 Update**  
    Thank you to everyone who attended the first session this morning and kicked off BGA23! Unfortunately (but luckily for the session attendees), five minutes after the session ended, the datacentre that hosts both [BlobToolKit](https://blobtoolkit.genomehubs.org) and [GoaT](https://goat.genomehubs.org) had a major cooling problem and all servers had to be shut down. We don't know when the servers will be back up, so we have postponed both sessions for now, and sent emails to all attendees.

    Sessions added/changed:

    - BlobToolKit: OLD TIME 4 Sep 20:00-21:00, **NEW TIME 11 Sep 17:00-18:00**
    - GoaT: OLD TIME 6 Sep 16:00-18:00, **NEW TIME 25 Sep 15:00-17:00**
    - "Aligning whole genomes using Cactus" new session added 29 Sep 16:00-18:00

!!! info "2023-09-20 Update"

    **Zoom Link** Click it to join the session. **All times are in UTC/GMT** and have a link which will add the event to your own preferred calendar in your time zone with the zoom link

    **Please join Discord** as zoom chats will be disabled. By now everyone should be on Discord, but if you're not, email contact [at] bga23 dot org for a link.

    **Please sign up for a Gitpod account** if the title below says GITPOD - instructions at [BGA23.org/gitpod](gitpod.md)

    **Session recordings at [youtube.com/@BGAcademy23](https://youtube.com/@bgacademy23)**. Some may take a day or two because YouTube takes time to provide an editable/edited version

| Title | Session Leader | Date and Time in UTC/GMT | 
|-------|----------------|--------------------------|
| [Checking for cobionts in public genomes using BlobToolKit](https://bga23.org/btk-view) | Sujai Kumar | 4 Sep 08:00-09:00<br/>~~4 Sep 20:00-21:00~~<br/>11 Sep 17:00-18:00 |
| [The TreeVal Pipeline: Generating evidence for manual curation - GITPOD](https://bga23.org/treeval-curation) | Genome Reference Informatics Team, Sanger | 8 Sep 11:00-13:00<br/>18 Sep 11:00-13:00 |
| [Assembling Mitogenomes from PacBio HiFi reads using MitoHiFi - GITPOD](https://bga23.org/MitoHiFi) | Marcela Uliano-Silva | 8 Sep 13:00-15:00 | 
| [Visualising genome assembly cobionts by running BlobToolKit locally - GITPOD](https://bga23.org/btk-cli) | Sujai Kumar, Rich Challis | 11 Sep 08:00-09:00<br/>11 Sep 20:00-21:00 |
| [Understanding k-mers and ploidy using Smudgeplot - GITPOD](https://bga23.org/smudgeplot) | Kamil Jaron | 12 Sep 14:00-16:00 |
| [Starting a comparative genome study from CNGBdb: datasets, analysis platform, and spatial transcriptomics database](https://bga23.org/cngbdb) | Xiaofeng Wei | 13 Sep 09:00-11:00 |
| [Searching and Downloading Genome Data using NCBI Datasets - GITPOD](https://bga23.org/ncbidatasets) | Nuala O'Leary, Mirian T. N. Tsuchiya, Eric Cox | 14 Sep 17:00-18:00 | 
| [Compare whole genome assembly alignments using NCBI‚Äôs Comparative Genome Viewer (CGV)](https://bga23.org/ncbi-cgv) | Sanjida Rangwala | 14 Sep 18:00-19:00 |
| [Genome profiling using Genomescope - GITPOD](https://bga23.org/genomescope) | Luc√≠a Campos-Dom√≠nguez, Kamil Jaron | 15 Sep 09:00-11:00 |
| [Introduction to Manual Curation - HiC and JBrowse - GITPOD](https://bga23.org/manual-curation) | Genome Reference Informatics Team, Sanger | 15 Sep 11:00-13:00<br/>19 Sep 11:00-13:00 |
| [OMA and OMArk for homology exploration and gene annotation quality control - GITPOD](https://bga23.org/oma-omark) | Yannis Nevers, Sina Majidian | 15 Sep 16:00-18:00<br/>21 Sep 08:00-10:00<br/> |
| [Introduction to NextFlow for Genomics - GITPOD](https://bga23.org/nextflow-genomics) | Solenne Correard | 19 Sep 19:00-21:00<br/>20 Sep 23:00-01:00<br/>26 Sep 16:00-18:00 |
| [BUSCO - from QC to gene prediction and phylogenomics - GITPOD](https://bga23.org/busco) | Rob Waterhouse | 20 Sep 08:00-10:00<br/>20 Sep 17:00-19:00 |
| [Classifying Transposable Elements with Earl Grey TE - GITPOD](https://bga23.org/earlgrey-te) | Tobias Baril | 21 Sep 12:00-14:00 |
| [Assembling Genomes with the VGP-Galaxy Pipeline](https://bga23.org/vgpgalaxy) | Delphine Lariviere | 21 Sep 17:00-19:00 |
| [Pangenome Assembly and Assessment using Minigraph and Bandage - GITPOD](https://bga23.org/pangenome){ : target="_blank" } | Zachary Cohen | 21 Sep 20:00-22:00 |
| [Gene prediction with BRAKER3 - GITPOD](https://bga23.org/B3) | Katharina Hoff | 22 Sep 08:00-10:00</br>25 Sep 12:00-14:00 |
| [Gfastar: a tool suite to assist genome assembly - GITPOD](https://bga23.org/gfastar) | Giulio Formenti, Cassidy Johnson | 22 Sep 14:00-16:00 |
| [Scalable telomere-to-telomere assembly with hifiasm - GITPOD](https://bga23.org/hifiasm-telomere) | Haoyu Cheng | 22 Sep 17:00-18:00 |
| [ENA Standards and Submission - GITPOD](https://bga23.org/ena-ebi-stands) | Maira Ihsan | 25 Sep 09:00-11:00<br/>26 Sep 18:00-20:00 |
| [Querying genome metadata and sequencing projects using GoaT](https://bga23.org/goat) | Cibele Sotero-Caio | ~~6 Sep 16:00-18:00~~<br/>25 Sep 15:00-17:00 |
| [ENA data retrieval and EMBL-EBI's Biodiversity Portals](https://bga23.org/enaebi-data/) | Alexey Sokolov, Joana Pauperio | 26 Sep 09:00-10:00<br/>27 Sep 18:00-19:00 |
| [Assess genome assemblies with Merqury - GITPOD](https://bga23.org/merqury) | Arang Rhie | 26 Sep 12:00-14:00<br/>[28 Sep 20:00-22:00](https://www.addevent.com/dir/?client=alorivImKzRucZeDgmNK105600&start=Thu+28+Sep+2023+20:00&end=Thu+28+Sep+2023+22:00&title=BGA23+Assess+genome+assemblies+with+Merqury&location=Online%2C+Zoom&timezone=Africa%2FMonrovia&organizer=Biodiversity+Genomics+Academy+2023&description=https%3A%2F%2Fsanger.zoom.us%2Fj%2F98392350855%3Fpwd%3DUTRzd1FNWlVSc0VKUTZ0ZHI5Qjl4QT09%5Cn%5Cnhttps%3A%2F%2FBGA23.org%2Fmerqury){ : target="_blank" }<br/>[Join Zoom](https://sanger.zoom.us/j/98392350855?pwd=UTRzd1FNWlVSc0VKUTZ0ZHI5Qjl4QT09){ : target="_blank" .md-button .md-button--primary } |
| [EASEL (Efficient, Accurate, Scalable Eukaryotic modeLs): software for eukaryotic structural and functional genome annotation](https://bga23.org/easel-annotation) | Jill Wegrzyn, Cynthia Webster, Vidya Vuruputoor, Karl Fetter | 27 Sep 13:00-15:00<br/>[29 Sep 03:00-05:00](https://www.addevent.com/dir/?client=alorivImKzRucZeDgmNK105600&start=Fri+29+Sep+2023+03:00&end=Fri+29+Sep+2023+05:00&title=BGA23+EASEL:+software+for+eukaryotic+structural+and+functional+genome+annotation&location=Online%2C+Zoom&timezone=Africa%2FMonrovia&organizer=Biodiversity+Genomics+Academy+2023&description=https%3A%2F%2Fsanger.zoom.us%2Fj%2F98516588524%3Fpwd%3DRkFWbzZrVHIrckdmbHlaMEJEK21Jdz09%5Cn%5Cnhttps%3A%2F%2FBGA23.org%2Feasel-annotation){ : target="_blank" }<br/>[Join Zoom](https://sanger.zoom.us/j/98516588524?pwd=RkFWbzZrVHIrckdmbHlaMEJEK21Jdz09){ : target="_blank" .md-button .md-button--primary } |
| [Annotating genomes the Ensembl way: General Concepts and Background + A Case study: Annotating Fungal Genomes - The Challenges!](https://bga23.org/ensembl-annotation) | Leanne Haggerty, Jos√© G P√©rez-Silva | 28 Sep 12:00-14:00 |
| [Annotating genomes the Ensembl way: Hands-on - From RNAseq reads to gene models - GITPOD](https://bga23.org/ensembl-annotation) | Leanne Haggerty, Jos√© G P√©rez-Silva | 28 Sep 14:00-16:00 |
| [Aligning whole genomes using Cactus - GITPOD](https://bga23.org/cactus) | Chiara Bortoluzzi, Thiago A.L. Genez | [29 Sep 16:00-18:00](https://www.addevent.com/dir/?client=alorivImKzRucZeDgmNK105600&start=Fri+29+Sep+2023+16:00&end=Fri+29+Sep+2023+18:00&title=BGA23+Aligning+whole+genomes+using+Cactus&location=Online%2C+Zoom&timezone=Africa%2FMonrovia&organizer=Biodiversity+Genomics+Academy+2023&description=https%3A%2F%2Fsanger.zoom.us%2Fj%2F98173725353%3Fpwd%3DZk9pcEl2VDF3OUlRRVEydld3Y09lUT09%5Cn%5Cnhttps%3A%2F%2FBGA23.org%2Fcactus){ : target="_blank" }[Join Zoom](https://sanger.zoom.us/j/98173725353?pwd=Zk9pcEl2VDF3OUlRRVEydld3Y09lUT09){ : target="_blank" .md-button .md-button--primary } |

## Calendar

<iframe src="https://calendar.google.com/calendar/embed?src=c_3245e1d36d7e1691a2271096ff59250ec2a69763ad15aa05c63854d0f3856cb7%40group.calendar.google.com&ctz=Europe%2FLondon" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>

## Proposed Sessions

Planning for Biodiversity Genomics Academy is over for this year.

However, the need for these types of sessions will not go away. If you want to volunteer to present future sessions like the ones above, or the proposed sessions below, or any others, please email contact [at] bga23.org.

| Topic |
|-------|
| Introduction to HiFi data (theory and practice) |
| Introduction to Hi-C data (theory and practice) |
| Introduction to Nanopore data (theory and practice) |
| Quality control of raw genomic data |
| Purging haplotypic duplication from assemblies (theory and practice) |
| YAHS and Hi-C scaffolding |
| Verkko: telomere-to-telomere assembly of diploid chromosomes |
| Using OrthoFinder to find orthologs across genomes |
| Using the Telomere Identification toolKit (tidk) |
| Using the Organelle Assembly ToolKit (oatk) |
